internal/containerupdate/exec_updater_test.go

```diff
@@ -101,6 +104,16 @@ func TestUpdateContainerRunsCommands(t *testing.T) {
 	}
 }
 
+func TestUpdateContainerRunsFailure(t *testing.T) {
```

Comment:
is it worth also testing that, if one run errors, the subsequent ones don't execute? (could write a new test, or make that happen with this test by having the FIRST exec call throw the error, and then assert on the number of exec calls).

Reply:
added!